prime-router/docs/standard-operating-procedures/update-observation-mapping-table.md

```diff
@@ -0,0 +1,217 @@
+The RCTC does a fairly good job of keeping up to date with LOINC and SNOMED codes and is regularly updated. It is anticipated that despite this there will be a requirement to map codes that are not present in the condition mapping table. These will have to be mapped manually after review by RS personnel in order to ensure that the proper condition code is mapped to the LOINC or SNOMED code. These codes can be submitted to CSTE valueset reviewers to be included in future releases. If a column is not applicable it can be left blank unless it is the "Code", "Code System", "Condition Name", "Condition Code"  "Value Source" columns.
+It is possible that an observation may not contain information intended to convey the results of a diagnostic test but instead represent additional important information regarding the testing sample of patient. Information of this type is referred to as "Ask at Order Entry" (AOE) questions commonly referred to by the abbreviation AOE only. The specific information being asked is identified in the observation with a LOINC code in OBX-3 (HL7-V2) or Observation.Code.Coding.Code (FHIR). We need to be able to identify these observations as AOEs in the same manner as we would identify which reportable condition the observation represents so that they can be filtered out if requested by STLTs and also to eliminate them as being unmapped observations. The LOINC codes for the base table come from two lists on LOINC.org.
+<br>1.) [LOINC terms for SARS-CoV-2 AOE questions](https://loinc.org/sars-cov-2-and-covid-19/)<br>
+2.) [Public health laboratory ask at order entry panel](https://loinc.org/81959-9)<br>
+
+
+## Updating RCTC valuesets
+
+The RCTC valuesets are updated regularly. These valuesets are stored in the Value Set Authority Center (VSAC) and can be accessed via an [API](https://www.nlm.nih.gov/vsac/support/usingvsac/vsacsvsapiv2.html).
+In order to update the table automatically using the VSAC API you will need a UMLS license (free). To obtain a liscense follow these steps: <br>
+
+1.) Visit the [VSAC Support Center](https://www.nlm.nih.gov/vsac/support/index.html)
+2.) Click on "Request a UMLS License" under "How to Use VSAC"
+3.) Login with an identity provider
+4.) Get apikey from profile to populate parameter -k in commands below
+    a.) Navigate to https://uts.nlm.nih.gov/uts/
+    b.) Select "My Profile" in top right
+    c.) Copy value from "API KEY"
+    
+
+### Updating all RCTC valuesets from the Value Set Authority Center (VSAC)
+Use ./prime lookuptables update-mapping CLI command to update all RCTC value sets from the VSAC. Be sure to select yes when prompted to write the table or output a new local csv.
+
+Example:
+```
+./prime lookuptables update-mapping -a -n observation-mapping -k YOUR_API_KEY -e prod
```

Comment:
Do we need to run `./prime login -e <env>` first before running these commands?

Reply:
Yes. I will add this

Reply:
added.